docs/AdvancedFeatures.md

```diff
@@ -150,4 +150,15 @@ An exception to this might be dynamic groups created and maintained by authors o
       allowExternalGroupNamesRegEx: external.* # the AC Tool groups can inherit from other external.* groups
 ```
 
+## Automatically purge obsolete groups
+The root element `obsolete_authorizables` can be used to automatically purge authorizables, that are not in use anymore:
```

Comment:
Yes it can also delete users as it is using purgeAuthorizables() under the hood...